src/uwtools/cli/run_forecast.py

```diff
@@ -60,11 +70,18 @@ def parse_args(args: List[str]) -> Namespace:
         metavar="FILE",
     )
     optional.add_argument(
-        "-m",
-        "--machine",
-        help="Path to YAML platform-definition file",
+        "--forecast-model",
+        choices={"FV3"},
+        help="The experiment to be run",
+    )
+    optional.add_argument(
+        "-o",
+        "--outfile",
```

Comment:
I might suggest that this should be `batch_script` instead of `outfile`. The latter is quite ambiguous when we're talking about running a whole task. It could refer to the model output, a log of some sort, etc.

Reply:
OK cool, we were discussing naming conventions and this was my placeholder, but that works